src/hadoop-resource-manager/deploy/hadoop-resource-manager.yaml.template

```diff
@@ -63,20 +63,22 @@ spec:
           value: {{ clusterinfo[ 'hadoopinfo' ][ 'hadoop_vip' ] }}
         - name: TIMELINE_SERVER_ADDRESS
           value: {{ clusterinfo[ 'hadoopinfo' ][ 'hadoop_vip' ] }}
+        - name: YARN_RESOURCEMANAGER_HEAPSIZE
+          value: "6144"
```

Comment:
have better raise the HEAPSIZE to 8G, change k8s resources limits to 10G

Reply:
Hi @mzmssg ,
Along currently testing, the resource manager it not the bottleneck.
So may we keep it as it is?